netgrasp/netgrasp.py

```diff
@@ -89,12 +89,12 @@ def _load_configuration(self):
         self.configuration = config.Config(self.debugger)
         
         # Load listen parameters.
-        self.listen["interface"] = self.configuration.GetText("Listen", "interface", False, False)
+        self.listen["interface"] = self.configuration.GetText("Listen", "interface", None, False)
         self.listen["active_timeout"] = self.configuration.GetInt("Listen", "active_timeout", 60 * 60 * 2, False)
         delay = self.configuration.GetInt("Listen", "delay", 15, False)
-        if (delay > 30):
```

Comment:
No worries, I totally understand. It's the C-heritage leaking through. ;-)

Thanks for the merge and all the feedback.